tiledb/cloud/file.py

```diff
@@ -36,6 +39,41 @@ def create_file(
         raise tiledb_cloud_error.check_exc(exc) from None
 
 
+def export_file_local(
+    uri: str,
+    output_uri: str,
+    timestamp: Optional[tuple] = None,
```

Comment:
Based on the docs it looks like this should be:

```suggestion
    timestamp: Union[Tuple[int, int], int, None] = None,
```

(also add `Union` and `Tuple` to the things you’re importing from `typing` above)